sql/core/src/test/scala/org/apache/spark/sql/streaming/TriggerAvailableNowSuite.scala

```diff
@@ -145,6 +146,12 @@ class TriggerAvailableNowSuite extends FileStreamSourceTest {
 
         val q = startQuery()
 
+        def makeDataFrame(a: Seq[Int]): DataFrame = if (SQLConf.get.ansiEnabled) {
+          a.map(_.toLong).toDF()
```

Comment:
Can we always use `.toLong` in this test?

Reply:
This is for making the expected answer of `Long union String`

Reply:
hmm, this streaming test wants to check `Long union String`?

Reply:
ok, let's update the test case instead